angularjs/angular-mocks.d.ts

```diff
@@ -39,8 +39,11 @@ declare module angular {
         dump(obj: any): string;
 
         // see http://docs.angularjs.org/api/angular.mock.inject
-        inject(...fns: Function[]): any;
-        inject(...inlineAnnotatedConstructor: any[]): any; // this overload is undocumented, but works
+        inject: {
+            (...fns: Function[]): any;
+            (...inlineAnnotatedConstructor: any[]): any; // this overload is undocumented, but works
+            strictDi(val: boolean): void;

```

Comment:
I think `val` is optional. https://github.com/angular/angular.js/blob/master/src/ngMock/angular-mocks.js#L2429